app/templates/summary.html

```diff
@@ -16,12 +16,13 @@
 
 {% if content.summary.summary_type == 'Summary' %}
 <div>
-  <h1 class="saturn">{{ _("Your answers") }}</h1>
+  <h1 class="saturn">{{ _("Check your answers and submit") }}</h1>
   <div class="print__message panel panel--simple panel--error u-mb-l">
     <h2 class="saturn">{{ _("Please remember to submit these answers") }}.</h2>
   </div>
-  <div class="print__hidden panel panel--simple panel--warn u-mb-l">
-    <h2 class="neptune">{{ _("Please check your answers carefully before submitting") }}.</h2>
+  <div class="print__hidden panel panel--simple panel--info u-mb-l">
+    <strong>{{ _("Please submit this survey to complete it") }}</strong>
+    <p>{{ _("You can check your answers below") }}</p>
```

Comment:
This change isn't explicitly detailed in the card, but i'm assuming it was desired?

Reply:
Yeah, that was the change in point 3 - It should look the same as the prototype linked

Reply:
point 3?

Reply:
From the PR description:
> 3. There are a couple of minor text changes to match the designs.

Reply:
ah, i was using the Trello card for reference